### DISCUSSION TITLE
Data Safety form of Sunbird SaaS application

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#660] [General Discussion]
We need to update the data safety form for the sunbird saas application.
There are few datatypes mention whether the app ask for this datatype. Here is the [link](https://support.google.com/googleplay/android-developer/answer/10787469?hl=en#types&zippy=%2Cdata-types) for the datatypes. 

What should i select here 
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/2932a9fc-646d-4e40-a1ad-6d44f674c686)

And if i select yes this are few questions listed.

![image](https://github.com/Sunbird-Ed/Community/assets/72926048/5538d823-ceed-46a5-95ae-c1f2cea5e9d6)
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/92af923d-e0da-4bb8-b64c-5f1dab75542f)
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/c5a56159-25ee-4f73-8ac3-cf8d58d2463d)
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/7cc2339e-787a-4a9f-b82f-bc357ca3a4ca)
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/eb7680f6-922d-4842-8a5a-3e7229713084)

Anyone who can help me with this 
@swayangjit @aishwaryashikshalokam @Prateek-slokam @varsha-sl @MeghnaSL @ashokreddy1208 @kameshbhr @mmanimegalai @rajeevsathish @chitranshu-keshav @anisha147 @vinukumar-vs @Krishnaj20 @kiranharidas187

**Comment by anisha147** [comment]
@gandham-santhosh Can you help here?

**Reply by gandham-santhosh** [reply]
@anisha147 this is specific to Mobile app. @swayangjit  please help here

**Reply by swayangjit** [reply]
@siddhishinde0723 This is specific to the app that you are customizing. In Sunbird point of view 

- Sunbird collects or shares any user type.
- Data is encrypted in transit.
- Account Creation methods username, password and other authentication.
- Approximate location is collected.
- Account deletion is supported

**Comment by anisha147** [comment]
Hi @siddhishinde0723 

We hope your queries are answered. If yes, can you please close this thread from your end?

cc: @mohitga @swayangjit
